multi-agent/lib/mas/blueprints/exceptions.py

```
@@ -50,6 +50,17 @@ def __init__(self, bad_keys: List[str]):
         super().__init__(f"Invalid metadata key(s): {bad_keys}")
 
 
+class BlueprintDuplicateNameError(BlueprintError):
+    """Raised when a blueprint name already exists for the same identity."""
+    def __init__(self, name: str):
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)